beluga_amcl/docs/PARAMETERS.md

```diff
@@ -0,0 +1,75 @@
+
+## Particle Filter Parameters
+
+| Parameter | Description | Navigation 2 AMCL | Beluga AMCL |
+|-----------|-------------|:-----------------:|:-----------:|
+| `max_particles` | Maximum allowed number of particles. | ✅ | ✅ |
+| `min_particles` | Minimum allowed number of particles. | ✅ | ✅ |
+| `pf_err` | Maximum particle filter population error between the true distribution and the estimated distribution. It is used in [KLD resampling](fox2001) to limit the allowed number of particles to the minimum necessary. | ✅ | ✅ |
+| `pf_z` | Upper standard normal quantile for P, where P is the probability that the error in the estimated distribution will be less than `pf_err` in [KLD resampling](fox2001). | ✅ | ✅ |
+| `spatial_resolution_[x, y, theta]` | Spatial resolution to create buckets for [adaptive KLD resampling][fox2001].                                | | ✅ |
+| `recovery_alpha_fast` | Exponential decay rate for the fast average weight filter, used in deciding when to recover from a bad approximation by adding random poses. | ✅ | ✅ |
+| `recovery_alpha_slow` | Exponential decay rate for the slow average weight filter, used in deciding when to recover from a bad approximation by adding random poses. | ✅ | ✅ |
+| `resample_interval` | Number of filter updates required before resampling. | ✅ | ✅ |
+| `selective_resampling` | When set to true, it will reduce the resampling rate when not needed and help avoid particle deprivation. The resampling will only happen if the effective number of particles $(N_{eff} = 1/ {\sum k_i^2})$ is lower than half the current number of particles. | ✅ | ✅ |
+| `update_min_a` | Rotational movement required from last resample for resampling to happen again. _Navigation 2 skips the filter update completely while Beluga only skips the resampling step. Sensor and motion updates are not skipped._ | ✅ | ✅ |
+| `update_min_d` | Translational movement required from last resample for resampling to happen again. _Navigation 2 skips the filter update completely while Beluga only skips the resampling step. Sensor and motion updates are not skipped._ | ✅ | ✅ |
+| `execution_policy` | Execution policy used to apply the motion update and importance weight steps to each particle [`seq`: sequential, `par`: parallel]. | | ✅ |
+
+## Motion Model Parameters
+
+| Parameter | Description | Navigation 2 AMCL | Beluga AMCL |
+|-----------|-------------|:-----------------:|:-----------:|
+| `robot_model_type` | Which odometry motion model to use [`differential_drive`, `omnidirectional_drive`, `stationary`]. _It also supports Navigation 2 plugin names [`nav2_amcl::DifferentialMotionModel` and `nav2_amcl::OmniMotionModel`] but it will load the equivalent Beluga model._ | ✅ | ✅ |
+| `alpha1` | Expected process noise in odometry’s rotation estimate from rotation for the differential and omnidirectional drive models. | ✅ | ✅ |
```

Comment:
@nahueespinosa nit: consider adding a reference to the literature describing these models in full.

Reply:
Added a reference for differential drive here https://github.com/Ekumen-OS/beluga/pull/196/commits/b0672a7827aa45034ea11d6fa2404de3df918006. AFAIK there is no literature describing the omnidirectional model.